docusaurus.config.js

```diff
@@ -7,6 +7,35 @@ if (!process.env.CI) {
     : 'fintan-contents/mobile-app-crib-notes';
 }
 
+const injector = (options) => {
+  const keys = Object.keys(options);
+  const placeHolders = keys.map((key) => new RegExp('{@inject: *' + key + '}'));
+  return inject;
+  function inject(tree) {
+    if (tree.type === 'root' || tree.type === 'element') {
+      tree.children = tree.children.map(inject);
+      if (tree.tagName === 'a' && tree.properties?.href) {
+        const href = decodeURI(tree.properties.href);
+        tree.properties.href = encodeURI(hasPlaceHolder(href) ? replace(href) : href);
```

Comment:
個人的には、こっちかなぁ、、、変更しない場合にわざわざdecode→encodeするのはちょっと怖い。

```suggestion
if (hasPlaceHolder(href)) {
  tree.properties.href = encodeURI(replace(href))
}
```